cmd/otelcol/config/collector/upstream_agent_config.yaml

```diff
@@ -164,30 +165,30 @@ service:
       - batch
       - resourcedetection
       #- resource/add_environment
-      exporters: [sapm, signalfx]
+      exporters: [otlp, signalfx]
       # Use instead when sending to gateway
-      #exporters: [otlp, signalfx]
+      #exporters: [otlp/gateway, signalfx]
```

Comment:
Why do we stick with these comments in every pipeline? Won't it be clearer to remove `otlp/gateway` and put this comment just in one place: `exporters::otlp`?

Reply:
Having two otlp exporters active but using only one of them doesn't seem right

Reply:
Submitted https://github.com/signalfx/splunk-otel-collector/pull/4587

Reply:
thanks